test/src/responder/component.test.ts

```diff
@@ -86,9 +86,12 @@ describe("ResponderAppointmentReducer", () => {
         await blockStore.start();
```

Comment:
Should really call blockStore.stop() somewhere

Reply:
Done in 4d9060a.